Nodejs/Product/Nodejs/Project/ProjectResources.cs

```diff
@@ -13,6 +13,7 @@ internal class SR : CommonSR
 
         internal const string NodeExeDoesntExist = "NodeExeDoesntExist";
         internal const string NodejsNotInstalled = "NodejsNotInstalled";
+        internal const string NpmIsInstalling = "NpmIsInstalling";
```

Comment:
>NpmIsInstalling [](start = 30, length = 15)

What are these for?

Reply:
This are the resource key, for the error message.

Reply:
It might be nice to use `nameof`.